python_flask/app/models/clients.py

```diff
@@ -16,20 +16,19 @@ class Client(db.Model):
     created_at = db.Column(db.DateTime)
     updated_at = db.Column(db.DateTime)
 
-    def __init__(self, name, address, city, state, zip, country, email,
-        contact, phone, created_at, updated_at):
-        self.name = name
-        self.email = email
-        self.address = address
-        self.city = city
-        self.state = state
-        self.zip = zip
-        self.country = country
-        self.email = email
-        self.contact = contact
-        self.phone = phone
-        self.created_at = created_at.utcnow()
-        self.updated_at = updated_at.utcnow()
+    def __init__(self, attribute):
```

Comment:
Minor nit, but this should probably be `attributes` not `attribute` because it isn't an attribute, but a dictionary containing multiple attributes.

Also, I wonder if there is a way to DRY this up. Seems awful boilerplatey, but I don't know Python well enough to have any concrete suggestions.